content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -0,0 +1,496 @@
+
+### Naming
+
+Determine the name of the clusters and networks in the mesh. These names will be used
+in the mesh network configuration and when configuring the mesh's service registries. 
+Assign a unique name to each cluster. The name must be a
+[DNS label name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names). 
+In the example below the main cluster is called `main0` and the remote cluster is `remote0`. 
+
+{{< text bash >}}
+$ export MAIN_CLUSTER_CTX=<...>
+$ export MAIN_CLUSTER_NAME=main0
+$ export REMOTE_CLUSTER_CTX=<...>
+$ export REMOTE_CLUSTER_NAME=remote0
+{{< /text >}}
+
+If the clusters are on different networks, assign a unique network name for each network. 
+{{< text bash >}}
+$ export MAIN_CLUSTER_NETWORK=network0
+$ export REMOTE_CLUSTER_NETWORK=network1
+{{< /text >}}
+
+If clusters are on the same network, the same network name is used for those clusters.
+{{< text bash >}}
+$ export MAIN_CLUSTER_NETWORK=network0
+$ export REMOTE_CLUSTER_NETWORK=network0
+{{< /text >}}
+
+## Deploy Istio in the main cluster
+
+Create the main cluster's configuration. Making the necessary changes based on the preparation above by replacing
+the v 
+
+{{< text yaml >}}
+cat <<EOF> istio-main-cluster.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      multiCluster:
+        clusterName: ${MAIN_CLUSTER_NAME}
+      network: ${MAIN_CLUSTER_NETWORK}
+
+# Mesh network configuration. This is optional and may be omitted if all clusters are on the same network.
+      meshNetworks:
+        ${MAIN_CLUSTER_NETWORK}:
+          endpoints:
+          # Always use Kubernetes as the registry name for the main cluster in the mesh network configuration
+          - fromRegistry: Kubernetes
+          gateways:
+          - registry_service_name: istio-ingressgateway.istio-system.svc.cluster.local
+            port: 443
+
+        ${REMOTE_CLUSTER_NETWORK}:
+          endpoints:
+          - fromRegistry: ${REMOTE_CLUSTER_NAME}
+          gateways:
+          - registry_service_name: istio-ingressgateway.istio-system.svc.cluster.local
+            port: 443
+
+  # Configure cross-cluster control plane access. Choose one of the three
+  # options below and delete the other two option's configuration.
+
+  # Option(1) - Use the existing istio-ingressgateway.
+      meshExpansion:
+        enabled: true
+
+  # Option(2) - Use a cloud provider’s load balancer.
+  # Change the Istio service `type=LoadBalancer` and add the cloud provider specific annotations.
+  # The example below shows the configuration with GCP/GKE.
+  components:
+    pilot:
+      k8s:
+        service:
+          type: LoadBalancer
+        service_annotations:
+          # See https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+          cloud.google.com/load-balancer-type: Internal
+
+  # Option(3) - Use a dedicated gateway
```

Comment:
It looks like there may be several issues making it difficult to create user gateways for this purpose: https://github.com/istio/istio/issues/21938, https://github.com/istio/istio/issues/21989, https://github.com/istio/istio/issues/21996. cc @ostromart , do you expect these to be fixed in 1.5.x? It isn't great to recommend an option that isn't actually possible yet.